core/src/main/scala/cats/data/Chain.scala

```diff
@@ -256,6 +256,93 @@ sealed abstract class Chain[+A] extends ChainCompat[A] {
     result
   }
 
+  /**
+   * take a certain amount of items from the front of the Chain
+   */
+  final def take(count: Int): Chain[A] = {
+    // invariant count >= 1
+    @tailrec
+    def go(lhs: Chain[A], count: Int, arg: Chain[A], rhs: Chain[A]): Chain[A] =
+      arg match {
+        case Wrap(seq) =>
+          if (count == 1) {
+            lhs.append(seq.head)
+          } else {
+            // count > 1
+            val taken = seq.take(count)
+            // we may have not takeped all of count
```

Comment:
hmm, out of curiosity, "takeped" – is it a typo or some slang I'm just not aware of? I also see "takeping" down below, so I guess it is intentional...

Reply:
no, I did `drop` first and I did find and replace to `take` and then changed the code. So, this was `dropped` and `dropping` with `s/drop/take/`.